pkg/machine/qemu/machine_windows.go

```diff
@@ -0,0 +1,24 @@
+package qemu
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+)
+
+func isProcessAlive(pid int) bool {
+	// TODO do we want /dev/null it?
```

Comment:
Will fix it. I believe it is better to add redirects for StdIn, StdOut and StdErr as we only support error codes.